common/c_cpp/src/c/mempool.c

```diff
@@ -76,6 +76,7 @@ memoryNode*
 memoryPool_addNode (memoryPool* pool, size_t nodeSize)
```

Comment:
Mempool mixed declaration changes are not relevant to this change - please remove from pull request and if necessary raise a separate pull request for that.